package.json

```diff
@@ -33,5 +33,8 @@
       "client",
       "types"
     ]
+  },
+  "msw": {
+    "workerDirectory": "public"
   }
-}
+}
```

Comment:
I'm not sure why this keeps happening -- There should be a new line at the end of this file. I think we might need to specify this in .editorconfig